# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.h
# Commit: a01eaeed432a
# Full Hash: a01eaeed432ab996a3f5275229be89814e92b95a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1708768
# File Overlap Count: 2
# Description:
#   Bug 1708768 - Preliminary cleanup of gfxPlatformFontList creation/initialization code; no functional change. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D114863
# ==============================================================================

diff -r 39c32384d7a9 -r a01eaeed432a gfx/thebes/gfxPlatform.h
--- a/gfx/thebes/gfxPlatform.h	Tue May 18 10:47:43 2021 +0000
+++ b/gfx/thebes/gfxPlatform.h	Tue May 18 11:10:42 2021 +0000
@@ -377,12 +377,7 @@
    * subclass). This function is responsible to create the appropriate subclass
    * of gfxPlatformFontList *and* to call its InitFontList() method.
    */
-  virtual gfxPlatformFontList* CreatePlatformFontList() {
-    MOZ_ASSERT_UNREACHABLE(
-        "oops, this platform doesn't have a "
-        "gfxPlatformFontList implementation");
-    return nullptr;
-  }
+  virtual bool CreatePlatformFontList() = 0;
 
   /**
    * Resolving a font name to family name. The result MUST be in the result of